### PR TITLE
Authenticated API: Allow signature creation for org with EU data processing consent required

### PR DESCRIPTION
**Pull request by wubini**
Deploy after code is deployed: https://github.com/controlshift/agra/pull/3694

### CODE REVIEW

**Comment by woodhull** [comment]
We should probably either write or link to a few paragraphs explaining all of this, what eu data processing consent means, and explain how to get a consent ID for use in the API. 

It probably also uncovers an improvement that should be made to the API here, we should probably get customers to setup an API opt in type, and use it here?

I think this might require a bigger architectural fix: essentially, how would an API consumer have any idea of what consent_content_version_id to use

File: source/includes/_authenticated_api_signatures.md [hunk 1]
```diff
@@ -85,14 +85,16 @@ For organisations that require email confirmation, a confirmation email will be
...
 | `signature[locale]`       | The language and localization setting of the member who has taken action. This is specified as an [ISO_639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) two-character language code combined with an optional [ISO 3166](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes) two character country code. For eg `en-US` or `en`. |
 | `signature[country]`      | The country specified as an [ISO 3166](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes) two character country code. This may be optional or required depending on your platform configuration. |
-| `signature[email_opt_in_type_id]` | The ControlShift email opt in type ID to use for this action. If this is not specified, fallbacks will be used, starting with any external opt in type that has been defined. |
+| `signature[email_opt_in_type_id]` | The External ID for the ControlShift email opt in type to use for this action. The email opt in type must have "external" context. You can list available email opt in types with [this endpoint](https://developers.controlshiftlabs.com/#email-opt-in-types)|
```
Comment (lavaturtle) [inline source/includes/_authenticated_api_signatures.md]: is `email_opt_in_type_id` right, or should it be `email_opt_in_type_external_id`?

**Review by lavaturtle — APPROVED** [review]
:+1: Looks good!

File: source/includes/_authenticated_api_signatures.md (hunk 1 above)
Comment (wubini) [reply source/includes/_authenticated_api_signatures.md]: good catch!

**Pull request merged**